client/app/src/main/res/layout/activity_my_book.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Do have a preference of using constraint vs  linear layout
how does this looks on bigger screens?
This is a layout comment so it isn't all that important

Reply:
Since it's constrained to bottom of the screen, the buttons remain the same distance as smaller screens. The distance between the end of the book detail component and the buttons enlarge on bigger screens.